src/views/Login.vue

```diff
@@ -1,40 +1,99 @@
 <template>
   <div class="container">
     <img src="../assets/wolox-logo.png" class="logo">
-    <div class="input-container">
-      <label for="name" class="label">First name</label>
-      <input id="name" class="input" v-model="firstName">
-    </div>
-    <div class="input-container">
-      <label for="name" class="label">Last name</label>
-      <input id="name" class="input" v-model="lastName">
-    </div>
-    <div class="input-container">
-      <label for="name" class="label">Email</label>
-      <input id="name" class="input" v-model="email">
-    </div>
-    <div class="input-container">
-      <label for="name" class="label">Password</label>
-      <input id="name" class="input" v-model="password" type="password">
-    </div>
-    <button :class="'primaryButton'" @click="printForm">Sign up</button>
+    <form @submit.prevent="onSubmit">
+      <!-- First Name -->
+      <div class="input-container">
+        <label for="name" class="label">First name</label>
+        <input
+          id="firstName"
+          v-model="$v.firstName.$model"
+          :class="['input', $v.firstName.$dirty && !$v.firstName.$error ? 'valid-input' : $v.firstName.$error ? 'error-input' : '']"
+        >
+      </div>
+      <!-- Last Name -->
+      <div class="input-container">
+        <label for="lastName" class="label">Last name</label>
+        <input
+          id="lastName"
+          v-model="$v.lastName.$model"
+          :class="['input', $v.lastName.$dirty && !$v.lastName.$error ? 'valid-input' : $v.lastName.$error ? 'error-input' : '']"
+        >
+      </div>
+      <!-- Email -->
+      <div class="input-container">
+        <label for="email" class="label">Email</label>
+        <input
+          id="email"
+          v-model="email"
+          :class="['input',
+            ($v.email.$invalid && submitted) || !$v.email.email
+              ? 'error-input'
+              : $v.email.email && !$v.email.$invalid ? 'valid-input' : '']"
+        >
+        <span class="error-label" v-if="!$v.email.email">El email no tiene un formato válido</span>
+        <span class="error-label" v-if="!$v.email.required && submitted">Campo requerido</span>
+      </div>
+      <!-- Password -->
```

Comment:
remove comment

Reply:
As it's a lot of code all together, I thought it was a good idea to separate each "section" with a comment.